### ISSUE TITLE
Zero amount transactions

### ISSUE THREAD

**Issue by pumpstation** [fireblocks/fireblocks-sdk-py#31]
There is following if construction in _create_transaction_ method:
```
if amount:
    body["amount"] = amount
```
In case of _amount=0_  (contract call for example), _bool(0)_ returns _False_, so amount field will be empty.

Better to use:
```
if amount is not None:
    body["amount"] = amount
```

**Comment by LukasBures** [comment]
I can confirm this issue. After the edit, it solved the mentioned problem.

**Comment by yarinvak** [comment]
This was fixed in v1.6.0
Thanks for reporting
